src/engine/source/hlp/src/LogQLParser.cpp

```diff
@@ -184,14 +188,14 @@ ExpressionList parseLogQlExpr(const char *expr)
 
                 if (!parseCapture(tokenizer, expresions))
                 {
-                    auto msg = "Invalid LogQL expression at '" + std::string(prev) + "'. Unable to parse capture";
-                    throw std::runtime_error(msg);
+                    auto msg = fmt::format("[HLP]Invalid LogQL expression at [{}]. Unable to parse capture expression.", std::string(prev));
+                    throw std::invalid_argument(msg);
```

Comment:
Let´s discuss in the team review which exception is expected here.
Because this parameter is truly a parsed value and maybe **invalid_argument** isn´t the correct exception to throw.